packages/now-node/src/index.ts

```diff
@@ -430,6 +430,14 @@ export async function startDevServer(
   opts: StartDevServerOptions
 ): Promise<StartDevServerResult> {
   const { entrypoint, workPath, config, meta = {} } = opts;
+
+  // Find the `tsconfig.json` file closest to the entrypoint file
+  const projectTsConfig = await walkParentDirs({
+    base: workPath,
+    start: join(workPath, dirname(entrypoint)),
+    filename: 'tsconfig.json',
+  });
```

Comment:
Let's add a test for this one. We already have one in `@now/node` but not in dev https://github.com/vercel/vercel/blob/be922796777a45cce5f075c7d73980e570058aef/packages/now-node/test/fixtures/18-nested-tsconfig/now.json

Reply:
f55560e